Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts

```diff
@@ -15,6 +15,6 @@
       <guid>1d9c2ddd-2cc3-4f63-bf25-884bde29c512</guid>
       <isReuseDriver>false</isReuseDriver>
       <isRun>true</isRun>
-      <testCaseId>Test Cases/Licensure Unit License Application/Submit appeal with invalid denial code from Public link</testCaseId>
+      <testCaseId>Test Cases/Licensure Unit License Application/Appeal/Submit appeal with invalid denial code from Public link</testCaseId>
```

Comment:
@fedecl-ot The "Appeal" folder should be on the Test Case folder, not inside the "Licensure Unit License Application"

Reply:
Done